testsuite/single_node_performance.py

```diff
@@ -216,7 +218,7 @@ class RunGroupConfig:
     RunGroupConfig(expected_tps=8000, key=RunGroupKey("resource-groups-global-write-and-read-tag1-kb", module_working_set_size=DEFAULT_MODULE_WORKING_SET_SIZE), included_in=Flow.RESOURCE_GROUPS, waived=True),
     RunGroupConfig(key=RunGroupKey("resource-groups-sender-write-tag1-kb"), included_in=Flow.CONTINUOUS | Flow.RESOURCE_GROUPS),
     RunGroupConfig(expected_tps=8000, key=RunGroupKey("resource-groups-sender-write-tag1-kb", module_working_set_size=DEFAULT_MODULE_WORKING_SET_SIZE), included_in=Flow.RESOURCE_GROUPS, waived=True),
-    RunGroupConfig(key=RunGroupKey("resource-groups-sender-multi-change1-kb"), included_in=LAND_BLOCKING_AND_C | Flow.RESOURCE_GROUPS),
+    RunGroupConfig(key=RunGroupKey("resource-groups-sender-multi-change1-kb"), included_in=LAND_BLOCKING_AND_C | Flow.RESOURCE_GROUPS | Flow.ADHOC),
```

Comment:
fa tests also failed 
```2025-01-07T04:07:01.5598573Z regression detected 29357.454824765642, expected median 40000, threshold: 32000.0), RunGroupKey(transaction_type='no_commit_apt-fa-transfer', module_working_set_size=1, executor_type='VM') didn't meet TPS requirements
2025-01-07T04:07:01.5601610Z regression detected 1810.3669920107332, expected median 40000, threshold: 32000.0), RunGroupKey(transaction_type='no_commit_apt-fa-transfer', module_working_set_size=1, executor_type='AptosVMSpeculative') didn't meet TPS requirements
```

Reply:
the "no_commit_" ones were not calibrated and has "waived=True"